requirements.txt

```diff
@@ -6,7 +6,9 @@
 #
 aiofiles==23.2.1
     # via -r requirements-build.in
-aiohttp==3.9.5
+aiohappyeyeballs==2.3.5
```

Comment:
What the heck?

Reply:
isn't naming fun...  [Happy Eyeballs](https://en.wikipedia.org/wiki/Happy_Eyeballs)